scripts/qa_crowbarsetup.sh

```diff
@@ -69,6 +69,7 @@ crowbar_install_log=/var/log/crowbar/install.log
 crowbar_init_api=http://localhost:4567/api
 crowbar_lib_dir=/var/lib/crowbar
 crowbar_api_v2_header="Accept: application/vnd.crowbar.v2.0+json"
+upgrade_progress_file=/var/lib/crowbar/upgrade/6-to-7-progress.yml
```

Comment:
Maybe, yes. But maybe we could add this once we have different possible values for the progress file